carforyou/carforyou-components-pkg
# CAR-8098: Integrate exhaustive-deps linting rule for pkgs in Components PKG

References [CAR-8098](https://autoricardo.atlassian.net/browse/CAR-8098)

## Motivation and context

We were working on the task to specify all missing useEffect dependencies. It is completed for listings and dealerhub. However, the rule is also important in our pkgs as well.


## Before

No linting rule and missing dependencies

## After

Linting rule and all dependencies are specified

## How to test

Browse through the apps where the `components-pkg` is used and make sure everything still works as expected. Keep an eye on:
- inputs
- modals
- intercom
